shuup/core/models/_suppliers.py

```diff
@@ -144,5 +150,11 @@ def update_stock(self, product_id):
     def update_stocks(self, product_ids):
         return self.module.update_stocks(product_ids)
 
+    def soft_delete(self):
+        if not self.deleted:
+            self.deleted = True
+            # Bypassing local `save()` on purpose.
```

Comment:
What would be the reasoning behind this?

Reply:
> What would be the reasoning behind this?

Sorry, don't understand what you mean.

Reply:
Just curios what this bypassing local save is for? I guess we could call `self.save(update_fields=("deleted",))`. Or even update the attr with queryset update `Supplier.objects.filter(id=self.pk).update(deleted=True)`. This might be just copy paste from `shuup.core.models.Product:soft_delete`, but maybe there is actual reason in product model which does not apply with supplier model.